src/item_group.cpp

```diff
@@ -197,6 +197,7 @@ Item_modifier::Item_modifier()
     , charges( -1, -1 )
```

Comment:
Here, `, dirt( 100, 9999 )` should be placed right before this line.

Reply:
OK, did it. Still not sure why the order matters here, but hopefully by tomorrow errors will be gone.